functions/vpn.bash

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
```

Comment:
Please be sure to follow all coding conventions with new contributions that have been outlined in `CONTRIBUTING.md` or that otherwise have been commonplace with the rewritten code. My future self will thank you greatly as it will save me a lot of work. :wink:

Reply:
sure but which ones was I missing here?

This wasn't meant to be reviewed yet but I forgot to set the Draft status, now I did.

Reply:
That's fine, I just wanted to leave a reminder to make sure to do a Quality Analysis before merging.